packages/vsce/__tests__/__e2e__/wiremock/__files/USSTree/zosmf-info.json

```diff
@@ -0,0 +1,85 @@
+{
+  "zos_version": "05.29.00",
+  "zosmf_port": "32070",
+  "zosmf_version": "29",
+  "zosmf_hostname": "localhost",
+  "plugins": [
```

Comment:
Are all these plugins necessary?? Can we return an empty array to reduce as much of the bloat as possible?